.gitattributes

```diff
@@ -0,0 +1,42 @@
+*.txt eol=lf
+*.gitignore text eol=lf
+*.sh text eol=lf
+*.md text eol=lf
+*.mod text eol=lf
+*.sum text eol=lf
+*.go text eol=lf
+*.yml text eol=lf
+*.proto text eol=lf
+*.json text eol=lf
+*.html text eol=lf
+*.svg text eol=lf
+*.js text eol=lf
+*.css text eol=lf
+*.yaml text eol=lf
+*.http text eol=lf
+*.ps1 text eol=lf
+*.g4 text eol=lf
+*.interp text eol=lf
+*.pem text eol=lf
+*.cnf text eol=lf
+*.conf text eol=lf
+*.gitmodules text eol=lf
+*.variants text eol=lf
+*.cmake text eol=lf
+*.bat text eol=lf
```

Comment:
should .bat (and .cmd) use crlf?

Reply:
had the same thought... i was in the midst of testing to make sure it'll still work when andrew merged. it seems like .bat at least doesn't care so it seems fine

Reply:
uh - he musta merged something else cause... i thought for sure this one was merged but it clearly says 'open'... so... yeah i'm just going blind it seems :(